phoebe/atmospheres/passbands.py

```diff
@@ -2816,7 +2816,7 @@ def update_passband(passband, local=True, content=None, gzipped=None):
     <phoebe.atmospheres.passbands.update_passband>.
 
     Download and install updates for a single passband from
-    http://tables.phoebe-project.org, retrieving
+    https://tables.phoebe-project.org, retrieving
```

Comment:
should we replace this with "passband server" or something like that because the url (while the default) can be configured?